src/Controls/tests/Core.UnitTests/BindableLayoutTests.cs

```diff
@@ -389,6 +389,77 @@ public void ValidateBindableProperties()
 			Assert.Equal(itemTemplateSelector, layout.GetValue(BindableLayout.ItemTemplateSelectorProperty));
 		}
 
+		[Fact]
+		public async Task DoesNotLeakChildListeningToBindingContextOnRemove()
+		{
+			var myViewModel = new MyViewModel();
+			var list = new ObservableCollection<MyViewModel> { myViewModel };
+			
+			var triggeredCount = 0;
+			var itemTemplate = new DataTemplate(() => new MyViewModelBoundComponent(onTextChangedCallback: () => triggeredCount++));
```

Comment:
Can we remove `triggeredCount` and `onTextChangedCallback` in these new tests? Nothing asserts against that value.

Reply:
The presence of that variable is what causes the leak, but this doesn't mean that the problem resides in the unit tests.
I guess it all depends on what the MAUI team says.